bartop-api/src/api/catalog/catalog.graphql.js

```diff
@@ -19,8 +20,63 @@ type ReplaceCatalogPayload {
   unchanged: Boolean
 }
 
+
+# Input type containing data needed to add a drink to a user's catalog
+input AddDrinkToCatalogInput {
+
+  # The unique ID of the user that corresponds to the catalog to be updated
+  userId: ID!
+
+  # The unique ID of the drink to be added to the catalog
+  drinkId: ID!
+}
+
+# Return type of addDrinkToCatalog
+type AddDrinkToCatalogPayload {
+
+  # The Drink object added to the catalog
+  drink: Drink
+
+  # A flag that will be 'true' if the given drink is already present in the catalog, and 'null' otherwise.
+  unchanged: Boolean
+
+  # A flag that will be 'true' if the given drink ID does not exist in the database, and 'null' otherwise
+  invalidDrink: Boolean
```

Comment:
Would it make more sense to have `Payload` types always have an `errors` key, which can contain an array of the different `error`s that were caused by the input? It's hard to find what the exact best practice is but I found some jawns about it:

* https://stackoverflow.com/questions/32662437/graphql-and-form-validation-errors
* https://medium.com/@tarkus/validation-and-user-errors-in-graphql-mutations-39ca79cd00bf
* https://stackoverflow.com/questions/35713889/handling-errors-in-mutations

It just feels weird to have an exact key in the payload that is just for one particular error (an invalid drink) vs. all payloads can have `errors`, and the client would be responsible for looking at the `errors` array and handling appropriately